api/users.go

```diff
@@ -35,16 +42,26 @@ type UsersAPI interface {
 	// Me returns actual user
 	Me(ctx context.Context) (*domain.User, error)
 	// MeUpdatePassword set password of actual user
-	MeUpdatePassword(ctx context.Context, password string) error
+	MeUpdatePassword(ctx context.Context, oldPassword, newPassword string) error
+	// SignIn signs in a user with username and password credentials. This overrides any previously set authentication token
```

Comment:
```suggestion
	// SignIn exchanges username and password credentials to establish an authenticated session with the InfluxDB server. The Client's authentication token is then ignored, it can be empty.
```